### ISSUE TITLE
Font size / cursor position bug

### ISSUE THREAD

**Issue by MaxwellVII** [arduino/Arduino#1908]
Little intro:
So I just got a lappy with 4k resoloution. The IDE defaults to 12pt font size, which is entirely unacceptable for a 15'' 4k monitor. So I go ahead and change the font size right?

Well, when you do that, the position of the cursor doesn't correctly correspond to where you are writing text. I've seen this happen on various other programs. You click at one point, and text starts emerging somewhere else on the line.

blehblehbleh[clicked here]blehbleh
ble[adding text here]hblehblehblehbleh

TL;DR: cursor doesn't correctly align on anything but 12pt font.

EDIT: Found that it only happens when changing the font size while you have code open

**Comment by ffissore** [comment]
This should be fixed with the latest hourly, which ships a new editor http://www.arduino.cc/en/Main/Software#hourly
Reopen if issue is still valid
